samples/DataGridListSample.js

```diff
@@ -87,8 +87,8 @@ enyo.kind({
 enyo.kind({
 	name: "moon.sample.GridSampleItem",
 	kind: "moon.GridListImageItem",
-	mixins: ["moon.SelectionOverlaySupport"],
-	selectionOverlayVerticalOffset: 35,
```

Comment:
Since another sample was created for BadgeOverlaySupport, this change should be reverted. As it stands now, this sample is broken because it has buttons for adjusting the selection options but no way to select an item.